pkg/cmd/server/start/start_master.go

```diff
@@ -242,10 +242,10 @@ func (o MasterOptions) RunMaster() error {
 			return err
 		}
 
-		if err := os.MkdirAll(path.Dir(o.MasterArgs.GetConfigFileToWrite()), os.FileMode(0755)); err != nil {
```

Comment:
I couldn't find anywhere to put this in docs, we don't actually document anywhere where non root users would interact with contents of /etc/origin so I've put a note in the release notes issue to mention it.